<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UndistortSequenceDialog</class>
 <widget class="QDialog" name="UndistortSequenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Undistort Sequence</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../calibrationtool_resources.qrc">
    <normaloff>:/images/resource-files/icons/XMALab-Icon.png</normaloff>:/images/resource-files/icons/XMALab-Icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Camera: </string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>346</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>347</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButtonAddVideo">
     <property name="text">
      <string>Add Video File</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="pushButtonAddFolder">
     <property name="text">
      <string>Add Image Folder</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pushButtonUndist">
     <property name="text">
      <string>Undistort</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../calibrationtool_resources.qrc"/>
 </resources>
 <connections/>
</ui>
